src/celeritas/phys/Interaction.hh

```diff
@@ -73,6 +73,7 @@ struct Interaction
 struct MscStep
 {
     bool      is_displaced{true}; //!< Flag for the lateral displacement
+    bool      geo_limited{false}; //!< Flag for the geometry limited step
```

Comment:
This isn't set by the `UrbanMscStepLimit` so doesn't need to be stored/retrieved in memory.

Reply:
Removed MscStep::geo_limited.